packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -310,10 +310,11 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
 
     if (this.props.onKeyDown) {
       this.props.onKeyDown(ev);
+    }
 
-      if (ev.isDefaultPrevented()) {
-        return;
-      }
+    // If the default has been prevented, do not process keyboard events.
+    if (ev.isDefaultPrevented()) {
+      return;
     }
```

Comment:
If the event was handled by a button inside of the focusZone and it preventDefault() and stopPropagation() does the event still get into this keyDown handler, it shouldn't... It should stop bubbling when stopPropagation() is called

Reply:
If you stopPropagation, it won't execute the focuszone's keydown.

If you preventDefault, it will execute it, but focuszone will not do the default logic.

Reply:
I *think* that is correct.

If you preventDefault in a div click handler inside an href, it will still execute the onClick of the href but won't nav to the link.